### ISSUE TITLE
Raw text data with boundaries of job descriptions

### ISSUE THREAD

**Issue by notani** [kris927b/SkillSpan#15]
Thank you for sharing this valuable resource! The repository currently contains CoNLL-format files, which appear to lack the boundaries between individual job descriptions or `SpaceAfter` information. Could you make the unannotated, raw text data with clear separations available?

**Comment by jjzha** [comment]
Hi Naoki,

Thanks a lot for your interest as well. I've now added a subdirectory `/json` in the data folder: https://github.com/kris927b/SkillSpan/tree/main/data/json

I have been mostly working with these files now. This contains the combined files for the conll files in json format. Each line indicates the `idx` of the sentence meaning it comes from the same job posting. Note that in train, dev, and test the `idx` counts restarts. Additionally `source` indicates what source it comes from (i.e., `house` or `tech`).

For dev, for some reason, I ended up with 58 total job postings, where this should have been 60. I can't explain this. Otherwise I hope this is helpful!
